Source/RPC/Structures/Version.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Version.swift
+//  CITA
+//
+//  Created by XiaoLu on 2019/4/22.
+//  Copyright © 2019 Cryptape. All rights reserved.
+//
+
+import Foundation
+
+public struct Version: Decodable {
+    public var softwareVersion: String
+
+    enum CodingKeys: String, CodingKey {
+        case softwareVersion
+    }
+
+    public init(from decoder: Decoder) throws {
```

Comment:
`Version` only contains a primitive `String` property. Do we really need to implement `init(from: Decoder)`?

Reply:
`{
    "jsonrpc":"2.0",
    "id":83,
    "result":{
        "softwareVersion":"v0.22.0"
    }
}`
This is the data structure, it's different from other data structure like
`
{
    "id": 83,
    "jsonrpc": "2.0",
    "result": "0x1d10"
}
`